yente/settings.py

```diff
@@ -148,6 +148,7 @@ def env_str(name: str, default: str) -> str:
 ES_PASSWORD = env_get("YENTE_ELASTICSEARCH_PASSWORD")
 ES_CLOUD_ID = env_get("YENTE_ELASTICSEARCH_CLOUD_ID")
 ES_SNIFF = as_bool(env_str("YENTE_ELASTICSEARCH_SNIFF", "false"))
+ES_CA_CERT = env_get("YENTE_ES_CERT_PATH")
```

Comment:
I know this is a bit over, but perhaps the env var could be `YENTE_ELASTICSEARCH_CA_PATH`?

Reply:
Yes! Way more consistant